tests/v3_api/test_upgrade.py

```diff
@@ -19,6 +19,11 @@
 validate_ingress = \
     ast.literal_eval(os.environ.get('RANCHER_INGRESS_CHECK', "True"))
 
+sshUser = os.environ.get('RANCHER_SSH_USER', "ubuntu")
+rancherVersion = os.environ.get('RANCHER_VERSION', "master")
```

Comment:
@bmdepesa Can we have this OS environment renamed to "RANCHER_SERVER_VERSION" ? We use this same name for test_deploy_rancher_server method in test_custom_host_reg.py.